.github/workflows/ci.yml

```
@@ -6,9 +6,21 @@ jobs:
+
+      - name: Lint + Format check
+        run: npx biome check .
+
+      - name: Type check
+        run: npx tsc --noEmit
+
+      - name: Build
+        run: npm run build
+
+      - name: Run tests
+        run: npm test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)